airmozilla/main/views.py

```diff
@@ -1560,6 +1560,15 @@ def event_livehits(request, id):
     return {'hits': total_hits}
 
 
+@json_view
+def event_status(request, id):
+    try:
+        event = Event.objects.get(pk=id)
+        return {'event_status': event.status}
+    except Event.DoesNotExist:
+        return {'event_status': ''}
```

Comment:
Let errors be errors :)
I would just let it be a plain 404 Not Found if someone supplied the wrong parameter. 
E.g.
```python
def event_status(request, id):
    return {'event_status': get_object_or_404(Event, id=id).status}
```